kong/plugins/prometheus/exporter.lua

```diff
@@ -60,9 +66,15 @@ local function init()
   metrics.memory_stats = memory_stats
 
   -- per service/route
-  metrics.status = prometheus:counter("http_status",
-                                      "HTTP status codes per service/route in Kong",
-                                      {"service", "route", "code"})
+  if ngx.config.subsystem == "http" then
+    metrics.status = prometheus:counter("http_status",
+                                        "HTTP status codes per service/route in Kong",
+                                        {"service", "route", "code"})
+  else
+    metrics.status = prometheus:counter("stream_status",
+                                        "Stream status codes per service/route in Kong",
+                                        {"service", "route", "code"})
```

Comment:
Do we have a well defined meaning of code in the stream subsystem?

Reply:
It uses a similar meaning from HTTP, but only part of them is meaningful (copied from http://nginx.org/en/docs/stream/ngx_stream_core_module.html):
```
session status (1.11.4), can be one of the following:
200
session completed successfully
400
client data could not be parsed, for example, the PROXY protocol header
403
access forbidden, for example, when access is limited for certain client addresses
500
internal server error
502
bad gateway, for example, if an upstream server could not be selected or reached.
503
service unavailable, for example, when access is limited by the number of connections
```